packages/design-systems/src/components/composite/SideBar.tsx

```diff
@@ -0,0 +1,172 @@
+import { TreeView } from "@ark-ui/react";
+import { Container } from "@components/patterns/Container";
+import { ChevronDownIcon } from "lucide-react";
+import { treeView } from "@tailor-platform/styled-system/recipes";
+
+export type SideBarItem = {
+  id: string;
+  label?: string;
+  isOpen?: boolean;
+  link?: React.ReactNode;
+  contents?: ContentItem[];
+};
+
+export type ContentItem = {
```

Comment:
型の実装はSideBarItemとほとんど重複しているので、以下にできないですかね?

contentsとchildrenは型の定義が同じなのでcontentsに統一しましょう。

```suggestion
export type ContentItem = SideBarItem & {
  isVisible?: Boolean;
}
```

Reply:
@IzumiSy 
こんな感じで修正しました。
[refactor(ds): fix props](https://github.com/tailor-platform/frontend-packages/pull/285/commits/b6a889af5db2f09d4b4ded738a9c6f0155b35ca7)

Reply:
`ContentItem` は `contents` と `children` の両方をフィールドでもってますが、これはまだ修正対応していないだけですかね?

Reply:
すみません、名前が違ったので、別々に考えていましたが、良く見たら型が一緒なので、省略できますね。。。
[refactor(ds): fix props](https://github.com/tailor-platform/frontend-packages/pull/285/commits/74f2b5e8c3429755106e7f8a09caa9c5f8736b64)

Reply:
@IzumiSy レビューお願いします！